app/views/registrations/new.html.erb

```diff
@@ -0,0 +1,82 @@
+          <%= t('.hey_make').html_safe %>
+        </h1>
+
+        <h3 id="diaspora-hearts">
+          <%= t('.diaspora') %>
+        </h3>
+
+        <h4 id="sign-up-text">
+          <%= t('.sign_up') %>
+        </h4>
+
+        <%= form_for(resource, :url => registration_path(resource_name), :html => {:class => "form-horizontal block-form", :autocomplete => "off"}) do |f| %>
+            <fieldset>
+              <div class="control-group">
+                <label class="control-label" for="user_email">
+                  <%= t('.email') %>
+		<small><%= t('.your_email_private') %></small>
+                </label>
+
+                <div class="controls">
+                  <%= f.email_field :email, :placeholder => "luke@hoth.net", :title => t('registrations.new.enter_email'), :required => true %>
+                </div>
+              </div>
+
+              <div class="control-group">
+                <label class="control-label" for="user_username">
+                  <%= t('.username') %>
+                </label>
+
+                <div class="controls">
+                  <%= f.text_field :username, :placeholder => "jedi_guy", :title => t('registrations.new.enter_username'), :required => true, :pattern => "[A-Za-z0-9_]+" %>
+                </div>
+              </div>
+
+              <div class="control-group">
+                <label class="control-label" for="user_password">
+                  <%= t('.password') %>
+                </label>
+
+                <div class="controls">
+                  <%= f.password_field :password, :placeholder => "••••••••", :title => t('registrations.new.enter_password'), :required => true, :pattern => "......+" %>
+                </div>
+              </div>
+
+              <div class="control-group">
+                <label class="control-label" for="user_password_confirmation">
+                  <%= t('.password_confirmation') %>
+                </label>
+
+                <div class="controls">
+                  <%= f.password_field :password_confirmation, :placeholder => "••••••••", :title => t('registrations.new.enter_password_again'), :required => true, :pattern => "......+" %>
+                </div>
+              </div>
+
+              <% if AppConfig.settings.captcha.enable? %>
+                <div class="control-group" id="captcha">
+                  <%= show_simple_captcha(:object => 'user', :code_type => 'numeric') %>
+                </div>
+              <% end %>
+              <%= invite_hidden_tag(invite) %>
+            </fieldset>
+            <% if AppConfig.settings.terms.enable? %>
+              <div id="terms" class="text-center">
+                <%= t('.terms', terms_link: link_to(t('.terms_link'), terms_path, target: "_blank")).html_safe %>
+              </div>
+            <% end %>
+            <%= f.submit t('.continue'), class: "new-btn", data: {disable_with: t('.submitting')} %>
+        <% end %>
+      </div>
+    </div>
+  </div>
+</div>
```

Comment:
I guess you started working without updating from upstream first, then rebased before opening the PR? #5391 was merged today and that ported this template to HAML